cv_generator/main/models.py

```diff
@@ -18,3 +18,17 @@ class WorkingExperience(models.Model):
     
     def __str__(self):
         return self.company_name + ' | ' + self.title
+
+
+class PersonalInformation(models.Model):
+    name = models.CharField(max_length=25, blank=False)
```

Comment:
```suggestion
    user = models.ForeignKey(User, on_delete=models.CASCADE)
    name = models.CharField(max_length=25, blank=False)
```
I think it makes sense to link the Personal Information to a specific user. We will be needing this later in order to make sure that logged users only see their information and data

Reply:
Yeah, that makes sense!